addons/l10n_in/models/res_partner.py

```diff
@@ -44,3 +44,9 @@ def onchange_vat(self):
     def _commercial_fields(self):
         res = super()._commercial_fields()
         return res + ['l10n_in_gst_treatment']
+
+    # this is test GST number use in EDI sandbox
```

Comment:
If you want to comment the method, explain it using a docstring instead of a comment outside the method scope

Reply:
okay